helm/alfresco-process-infrastructure/requirements.yaml

```diff
@@ -3,7 +3,7 @@ dependencies:
     repository: https://activiti.github.io/activiti-cloud-helm-charts
     version: 1.1.36
   - name: alfresco-identity-service
-    version: 1.1.1
+    version: 2.0.0
```

Comment:
2.0.0? Strange, shouldn't that be 1.2.x?

Reply:
actually that is the Identity-service chart version, and the keylock version under that is 1.2x